src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Schemas/ProductVendorType.cs

```diff
@@ -7,7 +7,7 @@ public class ProductVendorType: ExtendableGraphType<ExpProductVendor>
 {
     public ProductVendorType()
     {
-        Name = "Vendor";
+        Name = "ProductVendor";
```

Comment:
maybe use Core Vendor?
https://github.com/VirtoCommerce/vc-module-experience-api/blob/dev/src/VirtoCommerce.ExperienceApiModule.Core/Schemas/VendorType.cs

Reply:
Fixed. Next time, pleas ask to prevent duplication authors of original PRs: https://github.com/VirtoCommerce/vc-module-experience-api/pull/365